infor-design/enterprise-ng
# EP6453 - Added custom button option in searchfield

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Add custom button option in searchfield

The css isn't reflecting on NG but works in IDS so putting this in draft until then.

**Related github/jira issue (required)**:

Related to this:  https://github.com/infor-design/enterprise/pull/6691

**Steps necessary to review your pull request (required)**:

- Use the latest IDS version
- Pull this branch build and run the app
- Go to: http://localhost:4200/ids-enterprise-ng-demo/searchfield
- Click on the heart button, toast should appear
